index.html

```diff
@@ -532,8 +747,8 @@ <h2>
           If locking the orientation results in an orientation change, the
           promise will be resolved when the orientation will change as
           described in the <a href=
-          '#handling-screen-orientation-changes'>Handling screen orientation
-          changes</a> section.
+          '#screen-orientation-change-algorithm'>Screen orientation change
```

Comment:
```suggestion
          >Screen orientation change
```

Just a note: if you see `<a href="#whatever">` in a ReSpec document, that is usually an Editor bug. In ReSpec, all internal linking should occur automatically unless there is a really good reason not to.